versions.env

```diff
@@ -0,0 +1,51 @@
+# shellcheck disable=2034 # unused variable
```

Comment:
Can we must merge this into `variables.env`?

Reply:
yes, done